### ISSUE TITLE
2025-04-24, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#150]
# Today's Arxiv Papers

(1) [IberBench: LLM Evaluation on Iberian Languages](http://arxiv.org/abs/2504.16921v1), 2025-04-23

(2) [OptimAI: Optimization from Natural Language Using LLM-Powered AI Agents](http://arxiv.org/abs/2504.16918v1), 2025-04-23

(3) [Tracing Thought: Using Chain-of-Thought Reasoning to Identify the LLM Behind AI-Generated Text](http://arxiv.org/abs/2504.16913v1), 2025-04-23

(4) [Do Large Language Models know who did what to whom?](http://arxiv.org/abs/2504.16884v1), 2025-04-23

(5) [Enhancing Critical Thinking with AI: A Tailored Warning System for RAG Models](http://arxiv.org/abs/2504.16883v1), 2025-04-23

(6) [Context-Enhanced Vulnerability Detection Based on Large Language Model](http://arxiv.org/abs/2504.16877v1), 2025-04-23

(7) [Exploring How LLMs Capture and Represent Domain-Specific Knowledge](http://arxiv.org/abs/2504.16871v1), 2025-04-23

(8) [Planning with Diffusion Models for Target-Oriented Dialogue Systems](http://arxiv.org/abs/2504.16858v1), 2025-04-23

(9) [Emo Pillars: Knowledge Distillation to Support Fine-Grained Context-Aware and Context-Less Emotion Classification](http://arxiv.org/abs/2504.16856v1), 2025-04-23

(10) [Monte Carlo Planning with Large Language Model for Text-Based Game Agents](http://arxiv.org/abs/2504.16855v1), 2025-04-23

(11) [Improving Significant Wave Height Prediction Using Chronos Models](http://arxiv.org/abs/2504.16834v1), 2025-04-23

(12) [LRASGen: LLM-based RESTful API Specification Generation](http://arxiv.org/abs/2504.16833v1), 2025-04-23

(13) [GreenMind: A Next-Generation Vietnamese Large Language Model for Structured and Logical Reasoning](http://arxiv.org/abs/2504.16832v1), 2025-04-23

(14) [Process Reward Models That Think](http://arxiv.org/abs/2504.16828v1), 2025-04-23

(15) [LLM-assisted Graph-RAG Information Extraction from IFC Data](http://arxiv.org/abs/2504.16813v1), 2025-04-23

(16) [Decoupled Global-Local Alignment for Improving Compositional Understanding](http://arxiv.org/abs/2504.16801v1), 2025-04-23

(17) [MOOSComp: Improving Lightweight Long-Context Compressor via Mitigating Over-Smoothing and Incorporating Outlier Scores](http://arxiv.org/abs/2504.16786v1), 2025-04-23

(18) [Graph2Nav: 3D Object-Relation Graph Generation to Robot Navigation](http://arxiv.org/abs/2504.16782v1), 2025-04-23

(19) [How Effective are Generative Large Language Models in Performing Requirements Classification?](http://arxiv.org/abs/2504.16768v1), 2025-04-23

(20) [Lightweight Latent Verifiers for Efficient Meta-Generation Strategies](http://arxiv.org/abs/2504.16760v1), 2025-04-23

(21) [HEMA : A Hippocampus-Inspired Extended Memory Architecture for Long-Context AI Conversations](http://arxiv.org/abs/2504.16754v1), 2025-04-23

(22) [A Survey of AI Agent Protocols](http://arxiv.org/abs/2504.16736v1), 2025-04-23

(23) [IRIS: Interactive Research Ideation System for Accelerating Scientific Discovery](http://arxiv.org/abs/2504.16728v1), 2025-04-23

(24) [A Post-trainer's Guide to Multilingual Training Data: Uncovering Cross-lingual Transfer Dynamics](http://arxiv.org/abs/2504.16677v1), 2025-04-23

(25) [LLMCode: Evaluating and Enhancing Researcher-AI Alignment in Qualitative Analysis](http://arxiv.org/abs/2504.16671v1), 2025-04-23

(26) [ParetoHqD: Fast Offline Multiobjective Alignment of Large Language Models using Pareto High-quality Data](http://arxiv.org/abs/2504.16628v1), 2025-04-23

(27) [TIFIN India at SemEval-2025: Harnessing Translation to Overcome Multilingual IR Challenges in Fact-Checked Claim Retrieval](http://arxiv.org/abs/2504.16627v1), 2025-04-23

(28) [Cognitive Silicon: An Architectural Blueprint for Post-Industrial Computing Systems](http://arxiv.org/abs/2504.16622v1), 2025-04-23

(29) [Algorithmic Mirror: Designing an Interactive Tool to Promote Self-Reflection for YouTube Recommendations](http://arxiv.org/abs/2504.16615v1), 2025-04-23

(30) [Debunking with Dialogue? Exploring AI-Generated Counterspeech to Challenge Conspiracy Theories](http://arxiv.org/abs/2504.16604v1), 2025-04-23

(31) [Comparing Large Language Models and Traditional Machine Translation Tools for Translating Medical Consultation Summaries: A Pilot Study](http://arxiv.org/abs/2504.16601v1), 2025-04-23

(32) [Case Study: Fine-tuning Small Language Models for Accurate and Private CWE Detection in Python Code](http://arxiv.org/abs/2504.16584v1), 2025-04-23

(33) [PIS: Linking Importance Sampling and Attention Mechanisms for Efficient Prompt Compression](http://arxiv.org/abs/2504.16574v1), 2025-04-23

(34) [PsyCounAssist: A Full-Cycle AI-Powered Psychological Counseling Assistant System](http://arxiv.org/abs/2504.16573v1), 2025-04-23

(35) [Enhancing LLM-Based Agents via Global Planning and Hierarchical Execution](http://arxiv.org/abs/2504.16563v1), 2025-04-23

(36) [DTVM: Revolutionizing Smart Contract Execution with Determinism and Compatibility](http://arxiv.org/abs/2504.16552v1), 2025-04-23

(37) [Exploring human-SAV interaction using large language models: The impact of psychological ownership and anthropomorphism on user experience](http://arxiv.org/abs/2504.16548v1), 2025-04-23

(38) [Tinkering Against Scaling](http://arxiv.org/abs/2504.16546v1), 2025-04-23

(39) [6G EdgeAI: Performance Evaluation and Analysis](http://arxiv.org/abs/2504.16529v1), 2025-04-23

(40) [QuaDMix: Quality-Diversity Balanced Data Selection for Efficient LLM Pretraining](http://arxiv.org/abs/2504.16511v1), 2025-04-23

(41) [A Comprehensive Survey of Synthetic Tabular Data Generation](http://arxiv.org/abs/2504.16506v1), 2025-04-23

(42) [TraveLLaMA: Facilitating Multi-modal Large Language Models to Understand Urban Scenes and Provide Travel Assistance](http://arxiv.org/abs/2504.16505v1), 2025-04-23

(43) [Intelligent Depression Prevention via LLM-Based Dialogue Analysis: Overcoming the Limitations of Scale-Dependent Diagnosis through Precise Emotional Pattern Recognition](http://arxiv.org/abs/2504.16504v1), 2025-04-23

(44) [Amplified Vulnerabilities: Structured Jailbreak Attacks on LLM-based Multi-Agent Debate](http://arxiv.org/abs/2504.16489v1), 2025-04-23

(45) [Harden and Catch for Just-in-Time Assured LLM-Based Software Testing: Open Research Challenges](http://arxiv.org/abs/2504.16472v1), 2025-04-23

(46) [From Past to Present: A Survey of Malicious URL Detection Techniques, Datasets and Code Repositories](http://arxiv.org/abs/2504.16449v1), 2025-04-23

(47) [EMRModel: A Large Language Model for Extracting Medical Consultation Dialogues into Structured Medical Records](http://arxiv.org/abs/2504.16448v1), 2025-04-23

(48) [Private Federated Learning using Preference-Optimized Synthetic Data](http://arxiv.org/abs/2504.16438v1), 2025-04-23

(49) [Give LLMs a Security Course: Securing Retrieval-Augmented Code Generation via Knowledge Injection](http://arxiv.org/abs/2504.16429v1), 2025-04-23

(50) [Can Large Language Models Help Multimodal Language Analysis? MMLA: A Comprehensive Benchmark](http://arxiv.org/abs/2504.16427v1), 2025-04-23

(51) [Evaluating Multi-Hop Reasoning in Large Language Models: A Chemistry-Centric Case Study](http://arxiv.org/abs/2504.16414v1), 2025-04-23

(52) [Out-of-the-Box Conditional Text Embeddings from Large Language Models](http://arxiv.org/abs/2504.16411v1), 2025-04-23

(53) [Less is More: Enhancing Structured Multi-Agent Reasoning via Quality-Guided Distillation](http://arxiv.org/abs/2504.16408v1), 2025-04-23

(54) [EEmo-Bench: A Benchmark for Multi-modal Large Language Models on Image Evoked Emotion Assessment](http://arxiv.org/abs/2504.16405v1), 2025-04-23

(55) [ConTextual: Improving Clinical Text Summarization in LLMs with Context-preserving Token Filtering and Knowledge Graphs](http://arxiv.org/abs/2504.16394v1), 2025-04-23

(56) [SplitReason: Learning To Offload Reasoning](http://arxiv.org/abs/2504.16379v1), 2025-04-23

(57) [Text-to-TrajVis: Enabling Trajectory Data Visualizations from Natural Language Questions](http://arxiv.org/abs/2504.16358v1), 2025-04-23

(58) [Transitive Array: An Efficient GEMM Accelerator with Result Reuse](http://arxiv.org/abs/2504.16339v1), 2025-04-23

(59) [ClarifyCoder: Clarification-Aware Fine-Tuning for Programmatic Problem Solving](http://arxiv.org/abs/2504.16331v1), 2025-04-23

(60) [Media Content Atlas: A Pipeline to Explore and Investigate Multidimensional Media Space using Multimodal LLMs](http://dx.doi.org/10.1145/3706599.3720055), 2025-04-22

(61) [Capturing Symmetry and Antisymmetry in Language Models through Symmetry-Aware Training Objectives](http://arxiv.org/abs/2504.16312v1), 2025-04-22

(62) [Improving Automated Secure Code Reviews: A Synthetic Dataset for Code Vulnerability Flaws](http://arxiv.org/abs/2504.16310v1), 2025-04-22

(63) [The Paradox of Poetic Intent in Back-Translation: Evaluating the Quality of Large Language Models in Chinese Translation](http://arxiv.org/abs/2504.16286v1), 2025-04-22

(64) [Investigating LLMs in Clinical Triage: Promising Capabilities, Persistent Intersectional Biases](http://arxiv.org/abs/2504.16273v1), 2025-04-22

(65) [Learning Explainable Dense Reward Shapes via Bayesian Optimization](http://arxiv.org/abs/2504.16272v1), 2025-04-22

(66) [TeLLMe: An Energy-Efficient Ternary LLM Accelerator for Prefilling and Decoding on Edge FPGAs](http://arxiv.org/abs/2504.16266v1), 2025-04-22

(67) [FinNLI: Novel Dataset for Multi-Genre Financial Natural Language Inference Benchmarking](http://arxiv.org/abs/2504.16188v1), 2025-04-22

(68) [TTRL: Test-Time Reinforcement Learning](http://arxiv.org/abs/2504.16084v1), 2025-04-22

(69) [MR. Video: "MapReduce" is the Principle for Long Video Understanding](http://arxiv.org/abs/2504.16082v1), 2025-04-22

(70) [From Reflection to Perfection: Scaling Inference-Time Optimization for Text-to-Image Diffusion Models via Reflection Tuning](http://arxiv.org/abs/2504.16080v1), 2025-04-22

(71) [LLMs are Greedy Agents: Effects of RL Fine-tuning on Decision-Making Abilities](http://arxiv.org/abs/2504.16078v1), 2025-04-22

(72) [PHYBench: Holistic Evaluation of Physical Perception and Reasoning in Large Language Models](http://arxiv.org/abs/2504.16074v1), 2025-04-22

(73) [DATETIME: A new benchmark to measure LLM translation and reasoning capabilities](http://arxiv.org/abs/2504.16155v1), 2025-04-22

(74) [Automated Static Vulnerability Detection via a Holistic Neuro-symbolic Approach](http://arxiv.org/abs/2504.16057v1), 2025-04-22

(75) [Honey, I Shrunk the Language Model: Impact of Knowledge Distillation Methods on Performance and Explainability](http://arxiv.org/abs/2504.16056v1), 2025-04-22

(76) [Certified Mitigation of Worst-Case LLM Copyright Infringement](http://arxiv.org/abs/2504.16046v2), 2025-04-22

(77) [LLMs meet Federated Learning for Scalable and Secure IoT Management](http://arxiv.org/abs/2504.16032v1), 2025-04-22

(78) [LiveCC: Learning Video LLM with Streaming Speech Transcription at Scale](http://arxiv.org/abs/2504.16030v1), 2025-04-22

(79) [Benchmarking LLM for Code Smells Detection: OpenAI GPT-4.0 vs DeepSeek-V3](http://arxiv.org/abs/2504.16027v1), 2025-04-22

(80) [CAPO: Cost-Aware Prompt Optimization](http://arxiv.org/abs/2504.16005v2), 2025-04-22

(81) [Token-Aware Coding Flow: A Study with Nano Surge in Reasoning Model](http://arxiv.org/abs/2504.15989v1), 2025-04-22

(82) [From Human Memory to AI Memory: A Survey on Memory Mechanisms in the Era of LLMs](http://arxiv.org/abs/2504.15965v2), 2025-04-22

(83) [FairTranslate: An English-French Dataset for Gender Bias Evaluation in Machine Translation by Overcoming Gender Binarity](http://arxiv.org/abs/2504.15941v1), 2025-04-22

(84) [Low-Rank Adaptation of Neural Fields](http://arxiv.org/abs/2504.15933v1), 2025-04-22

(85) [Reasoning Physical Video Generation with Diffusion Timestep Tokens via Reinforcement Learning](http://arxiv.org/abs/2504.15932v1), 2025-04-22

(86) [StreamRL: Scalable, Heterogeneous, and Elastic RL for LLMs with Disaggregated Stream Generation](http://arxiv.org/abs/2504.15930v1), 2025-04-22

(87) [Language Models to Support Multi-Label Classification of Industrial Data](http://arxiv.org/abs/2504.15922v2), 2025-04-22

(88) [ViSMaP: Unsupervised Hour-long Video Summarisation by Meta-Prompting](http://arxiv.org/abs/2504.15921v1), 2025-04-22

(89) [Towards Test Generation from Task Description for Mobile Testing with Multi-modal Reasoning](http://arxiv.org/abs/2504.15917v1), 2025-04-22

(90) [Automated Bug Report Prioritization in Large Open-Source Projects](http://arxiv.org/abs/2504.15912v1), 2025-04-22

(91) [Synergizing RAG and Reasoning: A Systematic Review](http://arxiv.org/abs/2504.15909v1), 2025-04-22

(92) [Impact of Noise on LLM-Models Performance in Abstraction and Reasoning Corpus (ARC) Tasks with Model Temperature Considerations](http://arxiv.org/abs/2504.15903v1), 2025-04-22

(93) [SARI: Structured Audio Reasoning via Curriculum-Guided Reinforcement Learning](http://arxiv.org/abs/2504.15900v1), 2025-04-22

(94) [Dynamic Early Exit in Reasoning Models](http://arxiv.org/abs/2504.15895v1), 2025-04-22

(95) [Towards responsible AI for education: Hybrid human-AI to confront the Elephant in the room](http://arxiv.org/abs/2504.16148v1), 2025-04-22

(96) [Inducing Vulnerable Code Generation in LLM Coding Assistants](http://arxiv.org/abs/2504.15867v1), 2025-04-22

(97) [Exploring Cognitive and Aesthetic Causality for Multimodal Aspect-Based Sentiment Analysis](http://arxiv.org/abs/2504.15848v1), 2025-04-22

(98) [Pre-DPO: Improving Data Utilization in Direct Preference Optimization Using a Guiding Reference Model](http://arxiv.org/abs/2504.15843v1), 2025-04-22

(99) [DualOptim: Enhancing Efficacy and Stability in Machine Unlearning with Dual Optimizers](http://arxiv.org/abs/2504.15827v1), 2025-04-22

(100) [What's the Difference? Supporting Users in Identifying the Effects of Prompt and Model Changes Through Token Patterns](http://arxiv.org/abs/2504.15815v1), 2025-04-22
